### ISSUE TITLE
i128 feature without nightly.

### ISSUE THREAD

**Issue by rodrimati1992** [paholg/typenum#105]
Could you release a patch,or a minor version bump on `crates.io` to not require rust nightly when using the i128 feature?

[This line is the culprit:`#![cfg_attr(feature = "i128", feature(i128_type))]`
](https://docs.rs/typenum/1.10.0/src/typenum/lib.rs.html#48)

This is the error I get when trying to compile with that feature:
```
   Compiling type_level_values v0.0.21 (file:///home/matias/Documents/eclipse_workspace/type_level/type_level_values)
error[E0554]: #![feature] may not be used on the stable release channel
  --> /home/matias/.cargo/registry/src/github.com-1ecc6299db9ec823/typenum-1.10.0/src/lib.rs:48:31
   |
48 | #![cfg_attr(feature = "i128", feature(i128_type))]
   |                               ^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0554`.
error: Could not compile `typenum`.
warning: build failed, waiting for other jobs to finish...
error[E0554]: #![feature] may not be used on the stable release channel
  --> /home/matias/.cargo/registry/src/github.com-1ecc6299db9ec823/typenum-1.10.0/src/lib.rs:48:31
   |
48 | #![cfg_attr(feature = "i128", feature(i128_type))]
   |                               ^^^^^^^^^^^^^^^^^^^

```

**Comment by akhilles** [comment]
I believe this was addressed in the 1.11.0 release: https://github.com/paholg/typenum/commit/1f4bfb30d5a7defc16e9a43cae9d9c301a560768.

**Comment by Robbepop** [comment]
> I believe this was addressed in the 1.11.0 release: [1f4bfb3](https://github.com/paholg/typenum/commit/1f4bfb30d5a7defc16e9a43cae9d9c301a560768).

Indeed this can be closed. Works without `nightly` since 1.11.0.
